arch/x86/kernel/cpu/resctrl/ctrlmondata.c

```diff
@@ -522,12 +523,24 @@ int rdtgroup_schemata_show(struct kernfs_open_file *of,
 	return ret;
 }
 
+static int smp_mon_event_count(void *arg)
+{
+	mon_event_count(arg);
+
+	return 0;
+}
+
 void mon_event_read(struct rmid_read *rr, struct rdt_resource *r,
 		    struct rdt_domain *d, struct rdtgroup *rdtgrp,
 		    int evtid, int first)
 {
+	int cpu;
+
+	/* When picking a CPU from cpu_mask, ensure it can't race with cpuhp */
+	lockdep_assert_held(&rdtgroup_mutex);
```

Comment:
upstream code does not have this lockdep_assert_help. Reference (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=09909e098113bed99c9f63e1df89073e92c69891)

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev